renderer/components/UpdateBox.tsx

```diff
@@ -45,29 +44,32 @@ class UpdateBoxContainer extends React.Component<Props, State> {
   }
 
   chooseUpdate = () => {
-    dialog.showOpenDialog({
+    remote.dialog.showOpenDialog({
       filters: [
-        { name: 'Update Package', extensions: ['gz', 'tar.gz']}
-      ]
-    }, (filepaths: string[]) => {
-      if (filepaths === undefined) return;
-      this.setState({ updateFilepath: filepaths[0] });
+        { name: 'Update Package', extensions: ['zip']}
+      ],
+    }).then((dialogReturn: OpenDialogReturnValue) => {
+      if (dialogReturn.filePaths.length > 0) {
+        this.setState({ updateFilepath: dialogReturn.filePaths[0] });
+      }
     });
   }
 
   upgradeSoftware = () => {
     this.setState({ isUploading: true });
     const update = pathToName(this.state.updateFilepath);
```

Comment:
This appears to be unused, do we still want to keep it here? If not, we can just delete it.

Reply:
Yea, I'll delete